### DISCUSSION TITLE
Restore Window to Previous State Not Working with Shortcut Snapped Windows

### DISCUSSION THREAD

**Discussion by f-hossen** [rxhanson/Rectangle#1448] [Ideas]
As a daily user who loves this app, I would appreciate the addition of a feature where the screen restore function can recognise and revert to its previous state, even if windows were snapped into place using shortcuts. Currently, this feature only works with manually resized windows.

**Comment by rxhanson** [comment]
Thanks for the feedback! Indeed this is the intended functionality of "restore". I've intentionally kept Rectangle slim so it's more approachable, and I added the "last" action that I think you're looking for into [Rectangle Pro](https://rectangleapp.com/pro) (paid). With that said, I would still consider a pull request for adding this into Rectangle as long as it's configured via the Terminal and not in the UI.
